GliaWidgets/ViewController/EngagementViewController.swift

```diff
@@ -56,15 +56,9 @@ class EngagementViewController: ViewController, AlertPresenter {
             case .offerScreenShare(let conf, accepted: let accepted, declined: let declined):
                 self.offerScreenShare(with: conf, accepted: accepted, declined: declined)
             case .showEndButton:
-                let endEngagementButton = ActionButton(with: self.viewFactory.theme.chat.endButton)
-                endEngagementButton.tap = { viewModel.event(.closeTapped) }
-                view.header.setRightItem(endEngagementButton, animated: true)
+                view.header.showEndButton()
             case .showEndScreenShareButton:
-                let endEngagementButton = ActionButton(with: self.viewFactory.theme.chat.endButton)
-                endEngagementButton.tap = { viewModel.event(.closeTapped) }
-                let endScreenShareButton = HeaderButton(with: self.viewFactory.theme.chat.endScreenShareButton)
-                endScreenShareButton.tap = { viewModel.event(.endScreenSharingTapped) }
-                view.header.setRightItems([endScreenShareButton, endEngagementButton], animated: true)
+                view.header.showEndScreenSharingButton()
```

Comment:
It's very good that we removed the creation of buttons from the `bind(...)` method! It was really sticking out :)